docs/wiki/The context - CARML overview.md

```diff
@@ -0,0 +1,10 @@
+This section provides an overview of the Common Azure Resource Modules Library (CARML) repository.
```

Comment:
It seems a bit redundant to have this page & the 'The context' page. I'd suggest to merge the two

Reply:
The reason I kept them is I'd expect other sections to be added under the context, such as all the strategy topics we're discussing recently (personas, vision and mission etc)

Reply:
Gotcha. So I guess that turns it into a fundamtenal question whether the 'entry' page should only link to the different sub-pages or can have rich content itself, right?